modules/terms-and-conditions/modules/terms-and-conditions/index.js

```diff
@@ -0,0 +1,48 @@
+// import Icon from 'react-native-vector-icons/FontAwesome';
+import React, { useEffect, useState } from 'react';
+import { Text, View, ScrollView, TouchableOpacity, useWindowDimensions } from 'react-native';
+import { styles } from './styles';
+import HTML from "react-native-render-html";
+
+
+const TermsAndConditions = ({ navigation }) => {
+
+  const contentWidth = useWindowDimensions().width;
+  const [htmlContent, setHtmlContent] = useState('<h1> No Terms and Conditions Loaded </h1>');
+  
+  useEffect(() => {
+    //change the root url below to your project's url. 
+    fetch('https://<APP_URL_HERE>.botics.co/modules/terms/termsandconditions/')
+      .then(response => response.json())
+      .then(data => setHtmlContent(data[0]['body']))
+      .catch(err => alert("Terms and Conditions could not be loaded at this time."));
+
+  });
+
+
+  return (
+    <View
+      style={{
+        flex: 1,
+      }}>
+      <View style={styles.heading}>
+        <TouchableOpacity
+          style={{ padding: 5 }}
+          onPress={() => {
+            navigation.goBack();
+          }}>
+        </TouchableOpacity>
+        <Text style={styles.header}>TERMS AND CONDITIONS</Text>
+          <View></View>
+        </View>
```

Comment:
```suggestion
```
Empty View component and wrong closing jsx tag

Reply:
Good catch. I think the second closing tag is still necessary, though, to close the Header view.